### ISSUE TITLE
[Suggestion] button to auto dissable chrome bookmarks in new tab

### ISSUE THREAD

**Issue by Akorian** [zombieFox/nightTab#238]
**Problem**
I am always frustrated when I see my chrome bookmarks bar in the new tab. In the new tab, I don't need them. But I can't just deactivate them since I use them on all other sites.

**Idea for Solution**
I would like a button in the settings which if switched on, automatically hides the chrome bookmarks bar. But when you navigate into another tab, it activates the bookmark bar again. 

My **Attempt**
I wanted to code a greasy fork script to hide/unhide the bar depending on the tab title. But apparently, chrome doesn't allow scripts in stuff like the new tab.

**Screenshots for further explanation**
https://imgur.com/a/hIjJAF4

**Comment by Marcelx8** [comment]
Hi, I found a possible solution to remove the bookmarks bar permanently from outside Chrome. You would have to create a key in your Registry for this. Unfortunately, this removes the bookmarks always, until you remove the created key. So, even on your pages, you wish to see them you won't have them.

If you really would like to try it, I found an article with a guide on how to:
https://www.askvg.com/tip-how-to-hide-bookmarks-bar-on-new-tab-page-in-google-chrome/
